### DISCUSSION TITLE
I sponsored someone, but still not get the public Sponsor badge.

### DISCUSSION THREAD

**Discussion by daiwanxing** [community/community#59637] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

It's a bug?

**Comment by leeringrose** [comment, marked as answer]
Hey @daiwanxing , According to my understanding, one time sponsor couldn't get the badge, but for only ongoing sponsor.
Have a look at a link below for your reference.
https://github.com/orgs/community/discussions/18313

**Reply by daiwanxing** [reply]
Thanks reply
